website_update_email_notify/views/email_templates.xml

```diff
@@ -2,7 +2,7 @@
 <odoo noupdate="1">
     <record id="email_template_notify_email_change" model="mail.template">
         <field name="name">Email Changed</field>
-        <field name="email_to">${ctx['old_email']|safe}</field>
+        <field name="email_to">${ctx['old_email']|safe},${ctx['new_email']}</field>
```

Comment:
```suggestion
        <field name="email_to">${ctx['old_email']|safe},${ctx['new_email']|safe}</field>
```

or

```suggestion
        <field name="email_to">${ctx['old_email']|safe}</field>
        <field name="email_cc">${ctx['new_email']|safe}</field>
```

I might prefer the latter, since the primary purpose of the email should be to notify the old email address of the change in case of some security breach.

Reply:
Thank you. I agree new_email add in cc.
Confirmed work as expected.